woocommerce/woocommerce
# Add contributor command and auto tagging to release-post-generator

### All Submissions:






### Changes proposed in this Pull Request:

This introduces 3 changes to release post generator:

1. Restructure it for standalone nested commands (see README and testing instructions for example new commands)
2. Add a new contributors command that generates an HTML file with just contributors so you can add it to a draft release post.
3. Add post tagging that auto tags the release post with the correct basic tags. When we introduce beta and RC post commands or options these will be easy to modify, override, etc.


### How to test the changes in this Pull Request:

1. Run the new standalone command `pnpm run release-post contributors -- "6.8.0" "6.7.0"`
2. See that it outputs an HTML file with all the contributors in it.
3. Check that the modified command structure still works for the release post generator, e.g.: `pnpm run release-post release -- "6.8.0" "6.8.0" --outputOnly`

### Other information:




### FOR PR REVIEWER ONLY:
